### PR TITLE
Minor suggestions

### PR DESCRIPTION
**Pull request by wpotrzebowski**
We probably also need to cite Zenodo? 

```
@misc{https://doi.org/10.25495/7gxk-rd71,
  doi = {10.25495/7GXK-RD71},
  url = {https://www.zenodo.org/},
  author = {{European Organization For Nuclear Research} and {OpenAIRE}},
  keywords = {FOS: Physical sciences, Publication, Dataset},
  language = {en},
  title = {Zenodo},
  publisher = {CERN},
  year = {2013}
}
```

I've been also thinking if it would make sense to present/summarize recomendations as a checklist or table. It may help when if someone wants to e.g. come back to paper and quickly check what are recomendations. It can also be included in supplementary information.

### CODE REVIEW

File: src/tex/ms.tex [hunk 1]
```diff
@@ -138,7 +138,7 @@ \section{Prior}
         $d_h$/\si{\angstrom} & $10.0$ & $[8.0, 16.0)$\\
         $V_h$/\si{\angstrom^3} & $339.5$ & $[300.0, 380.0)$\\
         $d_t$/\si{\angstrom} & $21.0$ & $[10.0, 26.0)$\\
-        $\phi_t$ & $1.0$ & $[0.5, 1.0)$\\
+        $\phi_t$ & $1.0$ & $[0.5, 1.0]$\\
```
Comment (arm61) [inline src/tex/ms.tex:141]: Why this change? The [ ) syntax is used to show that the start point of the interval is included while the end point is not.

**Review by arm61 — CHANGES_REQUESTED** [review]

**Comment by arm61** [comment]
I never realised that Zenodo had its own reference!

File: src/tex/ms.tex (hunk 1 above)
Comment (wpotrzebowski) [reply src/tex/ms.tex:141]: I see. My point here is that in principle you shouldn't get value of 1.0 if it is `)` but I guess it is due to rounding? Maybe we can set it to e.g. 0.9 to avoid any confusion.

Comment (arm61) [reply src/tex/ms.tex:141]: `[0.5, 1.0)` indicates that the minimum is 0.5 and the maximum is 0.99999999... so that is accounting for the fact that 1.0 is impossible. While 0.9 removes any confusion the table reproduced from a table in another paper so it should match.
